### ISSUE TITLE
Mod Organizer 2+, Skyrim SE vs. CLARALUX mod

### ISSUE THREAD

**Issue by Elladdan** [ModOrganizer2/modorganizer#1035]
Hi,
I used Mod Organizer 2 for Skyrim SE. I properly installed mod CLARALUX SSE (nexus id 2371). This mod changes place of several (lighntning) objects. Cause it showed some bugs, I uninstalled it (totally remove from MO2). But changes that it did are still active in all skyrim games launched via MO2 - wrong places of several objects and their properties.  

I uninstalled every SkyrimSE mod from MO2, launched game but problem is still active in all saves of game.
I tried to start new clear game without mods from MO2 and I came to first place with wrong places and problem still active.
I closed MO2 and launched Skyrim from main game directory without mods and all is OK, no problems. 
I did it several times with different saves. Always when I launch game via MO2, I have some problem.
 
I know, this is only report, but I welcome every your advice.

Thanks

## Environment:
* Mod Organizer Version that exhibits the issue: v2.2.2.1
* Desktop OS/version used to run Mod Organizer: Windows 10

**Comment by LostDragonist** [comment]
Are the mod files (CLARALUX SSE.esp, CLARLUX SSE.bsa, etc.) present in the MO2 data tab?  If not, then the files are not present and it's basically impossible for MO2 to be the problem here.

Do you have any generated patches like DynDOLOD, Bashed Patch, Smashed Patch, etc.?  Those could be the issue.  If you don't know of anything like that, you'll just have to start disabling plugins/mods until things aren't broken anymore and that'll tell you which plugin/mod is causing the issue.

**Comment by Elladdan** [comment]
Thanks for reply. I wrote, claralux is completely out :) from computer
:). I have new game without all mods now. Result is: Same save launch
via MO2 shows mistakes, Same save launch from main game directory is
without problem. I tried it several times. I disabled every plugin and
mod. For example, I uninstalled all mods - from Skyrim SE main directory
I launch game, start new game and I go to first place where is problem
active. I saw nothing. I save game. then I game launch game via MO2,
open same save and problem is active. Then I launch game via MO2 with
disabled all mods and plugins, started new game and I go to first place
where is problem active and problem was active.  
Mod Claralux makes own settings directly in game. Maybe these it writes
anywhere. I spent many hours in installing mods and I wouldn't want next
one, but if will be necessary I will reinstall Skyrim, MO2 and all 263
mods and plugins:) But I need to know, whatever I have to clear.  
One again Thanks for help and interesting  
  


> ** Thu, 26 Mar 2020 00:02:45 -0700 - "ModOrganizer2/modorganizer"
<reply@reply.github.com> **  
  

>

>

>

> Are the mod files (CLARALUX SSE.esp, CLARLUX SSE.bsa, etc.) present in
the MO2 data tab? If not, then the files are not present and it's
basically impossible for MO2 to be the problem here.

>

> Do you have any generated patches like DynDOLOD, Bashed Patch, Smashed
Patch, etc.? Those could be the issue. If you don't know of anything
like that, you'll just have to start disabling plugins/mods until things
aren't broken anymore and that'll tell you which plugin/mod is causing
the issue.

>

> —  
You are receiving this because you authored the thread.  
Reply to this email directly, [view it on GitHub](https://github.com/Mod
Organizer2/modorganizer/issues/1035#issuecomment-604264675), or
[unsubscribe](https://github.com/notifications/unsubscribe-
auth/AJBPNAFJEWQESXCCIDLO65DRJL4ZLANCNFSM4LT767PQ).

**Comment by LostDragonist** [comment]
Did you also make sure the overwrite mod is empty?  If that still doesn't help, follow these instructions and maybe I can tell you something.

> Upload your log files here. Follow these steps:
> 
> 1) Go in the Settings, Diagnostics tab, set Log Level to Debug.
> 2) Restart MO.
> 3) Use the "Open Folders" button to open the logs folder.
> 4) Recreate your problem.
> 5) Close MO.
> 6) Drop mo_interface.log and the most recent usvfs log here. You might need to compress them.
> 
> Note: When you're done with fixing the problem, go set the Log Level back to Info.
> 
> ![](https://cdn.discordapp.com/attachments/458597400638914563/617049541350260794/openlogs.png)
>

**Comment by Elladdan** [comment]
OK... I tried it... I attached two pics... same save, same character,
same place and same version SSE :) I disabled all mods except Unofficial
patch SSE. I think this issue is rare for you too :)))  
  
  


> ** Thu, 26 Mar 2020 03:03:34 -0700 - "ModOrganizer2/modorganizer"
<reply@reply.github.com> **  
  

>

>

>

> Did you also make sure the overwrite mod is empty? If that still
doesn't help, follow these instructions and maybe I can tell you
something.

>

>> Upload your log files here. Follow these steps:

>>

>>   1. Go in the Settings, Diagnostics tab, set Log Level to Debug.

>>   2. Restart MO.

>>   3. Use the "Open Folders" button to open the logs folder.

>>   4. Recreate your problem.

>>   5. Close MO.

>>   6. Drop mo_interface.log and the most recent usvfs log here. You
might need to compress them.

>>

>> Note: When you're done with fixing the problem, go set the Log Level
back to Info.

>

> —  
You are receiving this because you authored the thread.  
Reply to this email directly, [view it on GitHub](https://github.com/Mod
Organizer2/modorganizer/issues/1035#issuecomment-604338011), or
[unsubscribe](https://github.com/notifications/unsubscribe-
auth/AJBPNADM3SLHI5XHO57QY2LRJMR7NANCNFSM4LT767PQ).

**Comment by Al12rs** [comment]
There are no logs here. It's highly unlikely this is a Mo2 problem so I'm closing for organizational purposes. Please join discord if you would like us to look into this
